pages/mesosphere/dcos/services/nifi/configure-dcos-access/index.md

```diff
@@ -2,7 +2,7 @@
 layout: layout.pug
 navigationTitle: Configuring DC/OS access for NiFi
 title: Configuring DC/OS access for NiFi
-menuWeight: 4
+menuWeight: 1010
```

Comment:
this change is to make `Configuring DC/OS access for NiFi` page to come at last option in dropdown menu